base/applications/drwtsn32/main.cpp

```diff
@@ -239,11 +263,12 @@ int __stdcall wWinMain(HINSTANCE, HINSTANCE, LPWSTR cmdLine, INT)
 
     TerminateProcess(data.ProcessHandle, data.ExceptionInfo.ExceptionRecord.ExceptionCode);
 
+    std::wstring FilenameW(Filename);
     std::string Message = "The application '";
     Message += data.ProcessName;
     Message += "' has just crashed :(\n";
     Message += "Information about this crash is saved to:\n";
-    Message += Filename;
+    Message += std::string(FilenameW.begin(), FilenameW.end());
```

Comment:
I'm not sure how this is going to work: does the UNICODE -> ANSI conversion works OK?

Reply:
This is one of the ways to convert wstring to string